screens/OnboardingScreen/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { StyleSheet } from "react-native";
+import Intro1 from "./Intro1";
+import Intro2 from "./Intro2";
+import Intro3 from "./Intro3";
+import Intro4 from "./Intro4";
+import { Button, Text, View } from "../../components/Themed";
+import Swiper from "react-native-swiper";
+import Colors from "../../constants/Colors";
+import { useMainNavigation } from "../../hooks/useMainNavigation";
+import { NavigationScreens } from "../../navigation/MainNavigationContext";
+
+export default function OnboardingScreen() {
+  const navigation = useMainNavigation();
+
+  return (
+    <View style={styles.swiper}>
+      <Swiper
+        loop={false}
+        index={0}
+        dotColor={"transparent"}
+        dotStyle={{ ...styles.dotStyle, ...styles.dotSizing }}
+        activeDotColor={Colors.allowedButtonColor}
+        activeDotStyle={styles.dotSizing}
+      >
+        <Intro1 />
+        <Intro2 />
+        <Intro3 />
+        <Intro4 />
+        {/*This is just a temporary measure to make sure we can go from onboarding to Day 1 episode creation */}
+        <View style={{ flex: 1 }}>
+          <Button
+            style={{ flex: 1, position: "absolute", top: 400 }}
+            onPress={() => {
+              navigation.navigate({ type: NavigationScreens.createEpisode });
+            }}
+          >
+            <Text style={{ flex: 1 }}>Go to episode creation.</Text>
+          </Button>
+        </View>
```

Comment:
Isn't there also the "Let's walk through this together screen"? I am unsure if that is part of this pr or not, but that would go here right?

Reply:
Yep, we were unsure if you and Iman had already merged that in, but the 4 intro slides should link to the tutorial

Reply:
I don't believe we had worked on that page. Just don't want it to slip through the cracks.

Reply:
It also looks like the onboarding photos are actually done now. So you probably could use them

Reply:
@dfarooq610 could you provide some clarification